libr/anal/p/anal_x86_cs.c

```diff
@@ -1013,8 +1013,17 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		{
 			char *src = getarg (&gop, 1, 0, NULL);
 			char *dst = getarg (&gop, 0, 1, "^");
-			esilprintf (op, "%s,%s,$z,zf,=,$p,pf,=,$s,sf,=,$0,cf,=,$0,of,=",
-				src, dst);
+			char *_dst;
```

Comment:
Why underline? I dont think we have any statement in the syntax doc about this. Better use tmp as varname